src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -269,7 +281,7 @@ const usePushNotificationHook = () => {
     return () => {
       PushNotification.unregister();
     };
-  }, [navigation, signedProfileId, localDb]);
+  }, [navigation, signedProfileId, localDb, currentChatScreen]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array of the `useEffect` hook has been updated to include `currentChatScreen`. This is a good practice as it ensures the effect runs whenever `currentChatScreen` changes, keeping the effect in sync with the state.

```diff
- }, [navigation, signedProfileId, localDb]);
+ }, [navigation, signedProfileId, localDb, currentChatScreen]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->